docs/content/_apidocs.mdx

```diff
@@ -48,6 +48,8 @@ APIs from the core `dagster` package, divided roughly by topic:
 
 - [Software-Defined Assets](/\_apidocs/assets) (Experimental). Software-defined assets sit on top of the graph/job/op APIs and enable a novel way of constructing Dagster jobs that puts assets at the forefront.
 
+- [Tables](/\_apidocs/tables) (Experimental) `TableRecord` and `TableSchema` objects can be attached to Dagster Types and events as metadata, where they are specially rendered in Dagit.
+
```

Comment:
would it make more sense to put this section under [`_apidocs/ops#event-metadata`](https://docs.dagster.io/_apidocs/ops#event-metadata), as opposed to a standalone page?

Reply:
You're right, I moved it.